src/dragdealer.js

```diff
@@ -84,6 +84,10 @@ var Dragdealer = function(wrapper, options) {
    *                      the handle to its closest step instantly, even when
    *                      dragging.
    *
+   *   - number speed=0.1: Speed can be set between 0 and 1, with 1 being the
+   *                       fastest. It represents how fast the handle will snap
```

Comment:
_how fast the handle will snap_ => _how fast the handle will **slide**_

The handle doesn't snap to a specific position when the instance doesn't have any _steps_